code-challenges.js

```diff
@@ -1,62 +1,126 @@
+// if (temps === 212) {
+//     console.log("212 is at boiling point.")
+// }
+const hotAndCold = (tempurature) => {
+    if (tempurature >= 212){
+        return "is at boiling point."
+}
+else if (tempurature >= 350){
+    return "is above boiling point."
+}
+else if (tempurature <= 42){
+    return "is below boiling point."
+}
+else {
+    return "error"
+    }
+}
+console.log(hotAndCold(42))
+console.log(hotAndCold(212))
+console.log(hotAndCold(350))
+// NOTES: The terminal is stating "ReferenceError: cannot access 'array' before initialization" - I'm not sure what I've done wrong - to generate the outputs I first tried "tempurature" instead of "hotAndCold" within the console.log but that also did not work.
+// // --------------------2) Create the code that will combine the two arrays and return the length using the test variables provided below.
+// const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
+// const padres1998WorldSeriesRuns = [6, 3, 5, 3]
+// // Expected output: 9
 // Pseudo code:
-
-// --------------------3) Create the code that will reverse the letters of a string using the test variable provided below.
-
-const currentCohort = "Foxtrot 2023"
+// name of function: competitions
+// input: 2 arrays of numbers
+// output: the amount of variables within both arrays combined
+// process: combine values within arrays and return the amount of characters of the combined values
+// const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
+// const padres1998WorldSeriesRuns = [6, 3, 5, 3]
+// const competitions = padres1984WorldSeriesRuns.concat.(padres1998WorldSeriesRuns){
+// }
+// console.log(competitions.length)
+// competitions.length
+var padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
+var padres1998WorldSeriesRuns = [6, 3, 5, 3]
+console.log(padres1984WorldSeriesRuns.concat(padres1998WorldSeriesRuns))
+    let array = [array.length]
+//NOTES: I keep getting an error message, I think I'm on the right track but there's probably a lot wrong with my syntax?
+// // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below.
+// const currentCohort = "Foxtrot 2023"
 // Expected output: "3202 tortxoF"
-
 // Pseudo code:
-
-// --------------------4) Create the code that will return the last index of the given value from the array using the test variables provided below.
-
+// name of function: currentCohort
+// input: 1 string
+// output: string written backwards
+// process: to seperate the string into seperate letters and then reverse the characters to create a version of the string that is written backwards.
+// const currentCohort = "Foxtrot 2023"
+// const reversedString = str.split(""){
+// }
+//     console.log(reversedString)
+var currentCohort = "Foxtrot 2023"
+console.log(currentCohort.split("").reverse)
+// NOTES: I'm not sure why the command .reverse isn't working. Again - most likely syntax issue? I thought that the command split and reverse would generate the correct output.
```

Comment:
You definitely have the right concept. Looks like the reverse was missing some ().
```js
console.log(currentCohort.split("").reverse())
```
This will give you an output of an array with the values in reversed. 
[
  '3', '2', '0', '2',
  ' ', 't', 'o', 'r',
  't', 'x', 'o', 'F'
]
Now find an array built-in method that will convert the array into a string so you can receive that expected output. Woohoo, you are very close.